packages/components/checkbox/src/checkbox-group.vue

```diff
@@ -54,11 +55,9 @@ const modelValue = computed({
   },
 })
 
-provide('CheckboxGroup', {
-  name: 'ElCheckboxGroup',
-  ...toRefs(props),
+provide(checkboxGroupContextKey, {
+  ...omit(props, ['modelValue', 'label', 'tag']),
```

Comment:
Writing like this will cause the attributes to lose reactive, [DEMO](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtYnV0dG9uIEBjbGljaz1cImNoYW5nZVwiPmNoYW5nZTwvZWwtYnV0dG9uPlxuICB7eyBzaXplIH19LXt7IGZpbGwgfX1cbiAgPGRpdiBjbGFzcz1cImRlbW8tYnV0dG9uLXN0eWxlXCI+XG4gICAgPGVsLWNoZWNrYm94LWdyb3VwIHYtbW9kZWw9XCJjaGVja2JveEdyb3VwXCIgOnNpemU9XCJzaXplXCI+XG4gICAgICA8ZWwtY2hlY2tib3ggdi1mb3I9XCJjaXR5IGluIGNpdGllc1wiIDprZXk9XCJjaXR5XCIgOmxhYmVsPVwiY2l0eVwiPlxuICAgICAgICB7eyBjaXR5IH19XG4gICAgICA8L2VsLWNoZWNrYm94PlxuICAgIDwvZWwtY2hlY2tib3gtZ3JvdXA+XG4gIDwvZGl2PlxuICA8ZGl2IGNsYXNzPVwiZGVtby1idXR0b24tc3R5bGVcIj5cbiAgICA8ZWwtY2hlY2tib3gtZ3JvdXAgdi1tb2RlbD1cImNoZWNrYm94R3JvdXBcIiA6c2l6ZT1cInNpemVcIj5cbiAgICAgIDxlbC1jaGVja2JveCB2LWZvcj1cImNpdHkgaW4gY2l0aWVzXCIgOmtleT1cImNpdHlcIiA6bGFiZWw9XCJjaXR5XCIgYm9yZGVyPlxuICAgICAgICB7eyBjaXR5IH19XG4gICAgICA8L2VsLWNoZWNrYm94PlxuICAgIDwvZWwtY2hlY2tib3gtZ3JvdXA+XG4gIDwvZGl2PlxuICA8ZGl2IGNsYXNzPVwiZGVtby1idXR0b24tc3R5bGVcIj5cbiAgICA8ZWwtY2hlY2tib3gtZ3JvdXAgdi1tb2RlbD1cImNoZWNrYm94R3JvdXBcIiA6c2l6ZT1cInNpemVcIiA6ZmlsbD1cImZpbGxcIj5cbiAgICAgIDxlbC1jaGVja2JveC1idXR0b24gdi1mb3I9XCJjaXR5IGluIGNpdGllc1wiIDprZXk9XCJjaXR5XCIgOmxhYmVsPVwiY2l0eVwiPlxuICAgICAgICB7eyBjaXR5IH19XG4gICAgICA8L2VsLWNoZWNrYm94LWJ1dHRvbj5cbiAgICA8L2VsLWNoZWNrYm94LWdyb3VwPlxuICA8L2Rpdj5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgeyByZWYgfSBmcm9tICd2dWUnXG4gIFxuY29uc3Qgc2l6ZSA9IHJlZignZGVmYXVsdCcpXG5jb25zdCBmaWxsID0gcmVmKCcjNDA5RUZGJylcbmNvbnN0IGNoZWNrYm94R3JvdXAgPSByZWYoWydTaGFuZ2hhaSddKVxuY29uc3QgY2l0aWVzID0gWydTaGFuZ2hhaScsICdCZWlqaW5nJywgJ0d1YW5nemhvdScsICdTaGVuemhlbiddXG5cbmZ1bmN0aW9uIGNoYW5nZSgpIHtcbiAgc2l6ZS52YWx1ZSA9IHNpemUudmFsdWUgPT09ICdkZWZhdWx0JyA/ICdsYXJnZScgOiAnZGVmYXVsdCdcbiAgZmlsbC52YWx1ZSA9IGZpbGwudmFsdWUgPT09ICcjNDA5RUZGJyA/ICcjNjdjMjNhJyA6ICcjNDA5RUZGJ1xufVxuPC9zY3JpcHQ+XG4iLCJQbGF5Z3JvdW5kTWFpbi52dWUiOiI8c2NyaXB0IHNldHVwPlxuaW1wb3J0IEFwcCBmcm9tICcuL0FwcC52dWUnXG5pbXBvcnQgeyBzZXR1cEVsZW1lbnRQbHVzIH0gZnJvbSAnLi9lbGVtZW50LXBsdXMuanMnXG5zZXR1cEVsZW1lbnRQbHVzKClcbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxBcHAgLz5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctOTU5NC1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmZ1bGwubWluLm1qc1wiLFxuICAgIFwiZWxlbWVudC1wbHVzL1wiOiBcInVuc3VwcG9ydGVkXCJcbiAgfVxufSIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHtcbiAgICBcInZ1ZVwiOiBcImh0dHBzOi8vdW5wa2cuY29tL0B2dWUvcnVudGltZS1kb21AbGF0ZXN0L2Rpc3QvcnVudGltZS1kb20uZXNtLWJyb3dzZXIuanNcIixcbiAgICBcIkB2dWUvc2hhcmVkXCI6IFwiaHR0cHM6Ly91bnBrZy5jb20vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL3ByZXZpZXctOTU5NC1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmZ1bGwubWluLm1qc1wiLFxuICAgIFwiZWxlbWVudC1wbHVzL1wiOiBcInVuc3VwcG9ydGVkXCIsXG4gICAgXCJAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZVwiOiBcImh0dHBzOi8vdW5wa2cuY29tL0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlQDIvZGlzdC9pbmRleC5taW4uanNcIlxuICB9LFxuICBcInNjb3Blc1wiOiB7fVxufSIsImVsZW1lbnQtcGx1cy5qcyI6ImltcG9ydCB7IGdldEN1cnJlbnRJbnN0YW5jZSB9IGZyb20gJ3Z1ZSdcbmltcG9ydCBFbGVtZW50UGx1cyBmcm9tICdlbGVtZW50LXBsdXMnXG5cbmxldCBpbnN0YWxsZWQgPSBmYWxzZVxuYXdhaXQgbG9hZFN0eWxlKClcblxuZXhwb3J0IGZ1bmN0aW9uIHNldHVwRWxlbWVudFBsdXMoKSB7XG4gIGlmIChpbnN0YWxsZWQpIHJldHVyblxuICBjb25zdCBpbnN0YW5jZSA9IGdldEN1cnJlbnRJbnN0YW5jZSgpXG4gIGluc3RhbmNlLmFwcENvbnRleHQuYXBwLnVzZShFbGVtZW50UGx1cylcbiAgaW5zdGFsbGVkID0gdHJ1ZVxufVxuXG5leHBvcnQgZnVuY3Rpb24gbG9hZFN0eWxlKCkge1xuICByZXR1cm4gbmV3IFByb21pc2UoKHJlc29sdmUsIHJlamVjdCkgPT4ge1xuICAgIGNvbnN0IGxpbmsgPSBkb2N1bWVudC5jcmVhdGVFbGVtZW50KCdsaW5rJylcbiAgICBsaW5rLnJlbCA9ICdzdHlsZXNoZWV0J1xuICAgIGxpbmsuaHJlZiA9ICdodHRwczovL3ByZXZpZXctOTU5NC1lbGVtZW50LXBsdXMuc3VyZ2Uuc2gvYnVuZGxlL2luZGV4LmNzcydcbiAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2xvYWQnLCByZXNvbHZlKVxuICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignZXJyb3InLCByZWplY3QpXG4gICAgZG9jdW1lbnQuYm9keS5hcHBlbmQobGluaylcbiAgfSlcbn0iLCJfbyI6eyJzaG93SGlkZGVuIjp0cnVlLCJzdHlsZVNvdXJjZSI6Imh0dHBzOi8vcHJldmlldy05NTk0LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzIn19)

Maybe use the `reactiveOmit` functions from @vueuse/core